spec/schedulers/AnimationFrameScheduler-spec.ts

```diff
@@ -20,6 +20,19 @@ describe('Scheduler.animationFrame', () => {
     }
   });
 
+  it('should act like the async scheduler if delay > 0', (done: MochaDone) => {
```

Comment:
I'd like to suggest to use `fakeTimer` same as other test case ([here](https://github.com/ReactiveX/rxjs/blob/e91d113e49acc9b0e12e2524040fa84cf3fa2c78/spec/schedulers/QueueScheduler-spec.ts#L11-L35)) to avoid intermittent test failure by timing condition of test execution environment (usually occurs on mobile emulator browser test)